docs/aurora/getting-started-on-aurora.md

```diff
@@ -151,13 +151,13 @@ Host github.com gitlab.com bitbucket.org
 	ProxyCommand /usr/bin/socat - PROXY:proxy.alcf.anl.gov:%h:%p,proxyport=3128
 ```
 
-If you need to use soemthing besides your default SSH key on Aurora for authentication to GitHub in conjunction with the above SSH workaround, you may set
+If you need to use something besides your default SSH key on Aurora for authentication to GitHub in conjunction with the above SSH workaround, you may set
 
 ```
-export GIT_SSH_COMMAND="ssh -i ~/.ssh/specialGitKey -F /dev/null"
+export GIT_SSH_COMMAND="ssh -i ~/.ssh/specialGitKey"
 ```
 
-where specialGitKey is the name of the private key in your `.ssh` directory, for which you have uploaded the public key to GitHub.
+where specialGitKey is the name of the private key in your `.ssh` directory, for which you have uploaded the public key to GitHub. The `-F` option can be used to specify a different ssh config file if needed.
```

Comment:
```suggestion
where specialGitKey is the name of the private key in your `.ssh` directory, for which you have uploaded the public key to GitHub. The `-F` option can be used to specify a different SSH config file if needed; for example, `-F none` will completely ignore your config file, including the above workaround. 
```

Reply:
I couldnt figure out who originally added this on Drupal--- does seem strange to include `-F /dev/null` since it would cancel the suggested workaround.